src/module/deleteTask.js

```diff
@@ -0,0 +1,14 @@
+import tasks from './taskStore.js';
+
+const deleteTask = (index) => {
+  const temp = tasks.filter((task) => task.index !== index);
+  tasks.length = 0;
+  let counter = 1;
+  temp.forEach((element) => {
+    element.index = counter;
```

Comment:
Nice work, but you could assign each element to its index by using index as the second parameter in the call back function

Reply:
- Thank you for your feedback. If you use as a second parameter, it is difficult to begin indexing at 1.